src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/TextSyndicationContentKindHelper.cs

```diff
@@ -1,23 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Generic;
-using System.Text;
-using System.Xml;
-using System.Xml.Schema;
-using System.Xml.Serialization;
-
-namespace System.ServiceModel.Syndication
-{
-    internal static class TextSyndicationContentKindHelper
```

Comment:
As this type needs to be brought back again, it would probably make sense to place it in the same file as the enum as that's the convention we generally follow.